app/src/main/java/com/jerboa/model/SiteViewModel.kt

```diff
@@ -54,30 +49,26 @@ class SiteViewModel(private val accountRepository: AccountRepository) : ViewMode
                     Log.d("Jerboa", "acc init for id: ${it.id}")
 
                     if (it.isAnon()) {
```

Comment:
Don't forget that you need to set a new auth when you change ur account/log out. Look in AccountViewModel (take over the design, I did with my pr) Encapsulate it so we dont have to worry with removeAccount and etc

Reply:
The `changeLemmyInstance` on L54 sets the auth.

Reply:
Ah yes this is a flow updates the api. mb